apiml-common/src/main/java/com/ca/mfaas/product/logging/annotations/InjectApimlLogger.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.product.logging.annotations;
+
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * After enabling apiml logging {@link EnableApimlLogger} it can be used as an injector.
+ * It injects {@link com.ca.mfaas.message.log.ApimlLogger} instance automatically with
```

Comment:
what's 'it' here? '{@link EnableApimlLogger}', or smth else?

Reply:
It is `InjectApimlLogger` class. the class that I give the information about

Reply:
See above, 'it' refers to InjectApimlLogger

Reply:
Then feel free to replace 'it' with InjectApimlLogger